content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -1,50 +1,102 @@
+   and click :guilabel:`Create and edit...` from the drop-down menu that appears.
 
-.. image:: manage/manage02.png
-  :align: center
+   When that's selected, a :guilabel:`Create Description Template` pop-up form appears, in which
+   the body of the description can be added, along with any attachments that may be necessary.
 
-Listing with item specifics
-===========================
+   .. image:: manage/create-description-template-popup-form.png
+      :align: center
+      :alt: The crete description template pop-up form in ebay tab on product form in Odoo sales.
 
-In order to add item specifics, you should create a product attribute with one
-value in the **Variants** tab on the product form.
+Listing (with product variants)
+===============================
 
-.. image:: manage/manage03.png
-  :align: center
+When the :guilabel:`Sell on eBay` feature is activated under the :guilabel:`eBay` tab on a product
+form for a product with variants, the available field configurations are the same as products
+without variants.
 
-Product Identifiers
+However, when the :guilabel:`Listing Type` us changed to :guilabel:`Fixed Price`, Odoo presents a
+variant table at the bottom of the :guilabel:`eBay` tab, in which the :guilabel:`Fixed Price` can be
+entered, the decision to :guilabel:`Publish on eBay` can be made for specific variants, along with
+other options.
+
+.. image:: manage/fixed-listing-price.png
+   :align: center
+   :alt: The fixed price listing type in the ebay tab on a product form in Odoo sales.
+
+Listing (with item specifics)
+=============================
+
+In order to add product specifics (e.g. brand names, etc.), a product attribute dedicated to that
+specific detail should be created -- with that detail represented by a single value -- in the
+:guilabel:`Attributes & Variants` tab of the product form.
+
+Open the desired product form, and click into the :guilabel:`Attributes & Variants` tab.
+
+Then, click :guilabel:`Add a line`, name the :guilabel:`Attribute` after the product detail (e.g.
+`Brand`), and enter the specific detail in the :guilabel:`Values` column (e.g. `Apple`).
+
+.. image:: manage/item-specific-listing-variants.png
+   :align: center
+   :alt: Sample of product specific variant listing for ebay purposes in Odoo sales.
+
+List item on eBay
+=================
+
+Once all configurations and customizations have been entered on the product form, click the
+:guilabel:`List Item on eBay` button to list that product on the synchronized eBay account.
+
+Or, click the :guilabel:`Link With Existing eBay Listing` to combine the product with an existing
+product listing on the synchronized eBay account.
+
+.. important::
+   Products can **only** be listed or linked to eBay if the proper configurations have been made to
+   synchronize an eBay account with Odoo. Check out the documentation on
+   :doc:`/applications/sales/sales/ebay_connector/setup` to learn more.
```

Comment:
Will not having this set up beforehand prevent them from completing any of the steps at the top of this doc? If so, this admonition would be helpful earlier on. If not, this is a good spot for it 👍

Reply:
This is a great point. However, I suggested such a thing in the newly-added intro sentence that you suggested previously, so I think this admonition serves as a decent bookend to the entire doc. What do you think?

Reply:
With an intro sentence at the top now I agree that this works best as a bookend here